service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsMapTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Map;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CaseDetailsMap.caseDetailsMap;
+
+class CaseDetailsMapTest {
```

Comment:
Missing the case when an existing key/value is overridden with a non null/emptyList?

Reply:
ManageHearingsController.handleAboutToSubmit

Reply:
Having to rely on a test of a class that uses the component is a bit difficult to trace and it breaks a bit the model of unit testing. If refactoring in the future, you removed ManageHearingsController the code branch would be uncovered. Not covering at unit level prevent you as well to make sure that you could actually use a mock() of that object. Happy to discuss the approach.

Reply:
sorry, I misunderstand you, though that you were asking where new class is used. Missing test  added